packages/block-editor/src/components/inserter/index.js

```diff
@@ -426,13 +424,7 @@ export const ComposedPrivateInserter = compose( [
 ] )( PrivateInserter );
 
 const Inserter = forwardRef( ( props, ref ) => {
-	return (
-		<ComposedPrivateInserter
-			ref={ ref }
-			{ ...props }
-			orderInitialBlockItems={ undefined }
-		/>
-	);
+	return <ComposedPrivateInserter ref={ ref } { ...props } />;
```

Comment:
We can probably follow up to remove the need for the private wrapper once (and if!) this PR is merged.

Reply:
I would keep the private wrapper. This shape allows for easy future private APIs without all the boilerplate being re-added.

Reply:
I was looking for why we haven't removed the private inserter. IMO, this doesn't seem like a good reason to keep things around just in case we might need them in the future.